### DISCUSSION TITLE
Uses to produce iOS apps, as Android has Termux to produce apps?

### DISCUSSION THREAD

**Discussion by ETERNALBLUEbullrun** [ish-app/ish#2388] [Ideas]
Must know to figure out whether or not to purchase a device with iOS.
`ish` emulates `Linux`, so know is possible to install `g++` to compile programs to run through the emulated `Linux`,
but does `ish` also allow the stuff required to produce usable apps for iOS, as Android's `Termux` does**?
Android has so much problems with security/stability that it almost does not have any uses at all,
but if the design of iOS makes it impossible to produce apps for iOS,
iOS is just a waste of money too.

** https://gist.github.com/tmatz/817bf03433e059bf89c63dc33f286ccb#file-build_android_app_on_termux-md

**Comment by itskenny0** [comment]
You can't sideload apps on iOS that weren't signed. Don't think you can sign one on device. Unless you're jailbroken, in which case you can install anything.

**Comment by ETERNALBLUEbullrun** [comment]
Can use this for Xcode? https://github.com/kholia/OSX-KVM
How about AWS? "There are cloud Mac providers now - AWS are trialing M1 Mac instances." https://aws.amazon.com/ec2/instance-types/mac/
If can not produce apps for iOS, can at least produce apps for OSX?
